libs/external-router/README.md

```diff
@@ -28,107 +21,11 @@ Install with yarn:
 yarn add @daffodil/external-router
 ```
 
-## Usage
```

Comment:
I think we should keep the getting started inside the overview.

Reply:
I disagree. The getting started portion in the Usage page is directly related on how to use the package. It would be disjointed to move it out of usage where usage only has configurations.